t/01-distributed.t

```diff
@@ -0,0 +1,205 @@
+#!perl
+use 5.006;
+use strict;
+use warnings;
+use Test::More;
+use Test::Exception;
```

Comment:
I'd always recommend [Test::Fatal](https://metacpan.org/pod/Test::Fatal) over [Test::Exception](https://metacpan.org/pod/Test::Exception), the latter depends on [Scope::Upper](https://metacpan.org/pod/Scope::Upper) which does some strange things with scopes. Also, this isn't listed in `cpanfile`.